flutter/ARCHITECTURE.md

```diff
@@ -122,16 +106,13 @@ making things slower just to follow some principles that don't necessarily apply
 not make sense (or even be completely dumb) for some, but may be good for others.
 [Relevant xkcd](https://xkcd.com/927/).
 
-One extra thing: this is heavily influenced by a bunch personal opinion and experiences in some production projects
-that the team has worked on. This project's external dependencies will keep changing as the time goes on, Flutter will
-also keep evolving, and we have to adapt in a way to maintain consistency, integrity and scalability of our solution.
-So, it's probable that there is (or will be) better ways to achieve the same goals/objectives, and for this, we look
-into your help to make this project's architecture continuously provide a good developer experience to add new features,
-update old ones and keep those nasty bugs away.
+Please, keep the [General ARCHITECTURE.md](../ARCHITECTURE.md#one-last-thing) introduction in mind - there are a ton of
+personal opinions and experiences here. This means that there **will be better ways to build what we've done**, and it's
+what we are aiming for: your help to improve this architecture over the time.
```

Comment:
NIT: Same as https://github.com/olmps/memo/pull/194/files#r716847997

Reply:
Your link leads to no specific comment for some reason. Anyways, I've answered them there.

Reply:
I was trying to reference this comment: https://github.com/olmps/memo/pull/194#discussion_r716809096, but that's just me personal opinion, I have no arguments why it would be better than yours writing style, so let's move on